src/Build/BackEnd/Components/SdkResolution/SdkResolverService.cs

```diff
@@ -425,6 +498,12 @@ private SdkResult ResolveSdkUsingAllResolvers(int submissionId, SdkReference sdk
                 results.Add(result);
             }
 
+            if (IsUnneededWorkloadSdk(sdk, solutionPath, projectPath))
+            {
+                sdkResult = (SdkResult)resultFactory.IndicateSuccess(null, null, null);
```

Comment:
it's good to specify the argument name : when passing null literals (same for true/false/0 etc), so we can see what is being passed that is null